src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -498,11 +498,12 @@ private EnumValueDefinition ToEnumValue(GraphQLEnumValueDefinition valDef)
 
         protected virtual QueryArgument ToArgument(GraphQLInputValueDefinition inputDef)
         {
-            return new QueryArgument(ToGraphType(inputDef.Type))
+            var graphType = ToGraphType(inputDef.Type);
```

Comment:
Is something special here in moving this line from property initialization? I see another usage of `  ResolvedType = ToGraphType(inputDef.Type),` in ` protected virtual FieldType ToFieldType(string parentTypeName, GraphQLInputValueDefinition inputDef)`.

Reply:
The old code called ToGraphType twice; I reduced it to one call.  However, we can put it back in the constructor and eliminate setting `ResolvedType` (since it's already set within the constructor), if you like.

I really don't know anything about this code; I just fixed the default value handling.

Reply:
🤦 I see now, my bad. Just remove line with property initialization and return back `return new QueryArgument(ToGraphType(inputDef.Type))`.

Reply:
See two code suggestions below; merge them both if you like and it should compile and test properly.